dod-advana/gamechanger-web
# Task/document usage analyzer

## Description
Task: Document Usage Analyzer - added a new table describing documents, how many times they've been opened, how many people have viewed them, and what searches were used to get to that document

## Related Issue/Ticket

https://jira.di2e.net/browse/UOT-114867

## Testing instructions

- navigate to admin -> search pdf mapping -> bottom table
- take a look at the columns, sort them by different columns, try the filters
- make a new (unique is easier) search, and see if you can find what you searched on the table. 

## Testing Suite Updates

appStatsController.test.js
